src/index.js

```diff
@@ -0,0 +1,22 @@
+import './style.css';
+
+const testList = [
+  { task: 'Do dishes', completed: false, index: 1 },
+  { task: 'Do laundry', completed: false, index: 3 },
+  { task: 'Walk dog', completed: false, index: 2 },
+];
+
+function displayList(l) {
```

Comment:
On page load, the task array is not rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task. Please make the corresponding changes to make your list of tasks be rendered.

Check the image below for a better understanding:
![Capture2](https://user-images.githubusercontent.com/75709904/149545522-2d01076e-d46f-4795-a4b2-bcff53760ead.PNG)

Reply:
**If I follow these steps it looks ok to me:**

- git clone https://github.com/gdumani/todolist.git
- cd todolist
- git checkout develpment
- npm install
- npm run start

I get this image
![image](https://user-images.githubusercontent.com/60476910/149549867-8b7cc7ab-e70c-475e-ad3d-2c5e73c157ca.png)